src/Promitor.Agents.Scraper/Validation/Steps/Sinks/AtlassianStatuspageMetricSinkValidationStep.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Promitor.Agents.Core.Validation;
+using Promitor.Agents.Core.Validation.Interfaces;
+using Promitor.Agents.Core.Validation.Steps;
+using Promitor.Agents.Scraper.Configuration;
+using Promitor.Core.Scraping.Configuration.Providers.Interfaces;
+
+namespace Promitor.Agents.Scraper.Validation.Steps.Sinks
+{
+    public class AtlassianStatuspageMetricSinkValidationStep : ValidationStep,
+        IValidationStep
+    {
+        private readonly IMetricsDeclarationProvider _metricsDeclarationProvider;
+        private readonly IOptions<ScraperRuntimeConfiguration> _runtimeConfiguration;
+
+        public AtlassianStatuspageMetricSinkValidationStep(IOptions<ScraperRuntimeConfiguration> runtimeConfiguration, IMetricsDeclarationProvider metricsDeclarationProvider, ILogger<AtlassianStatuspageMetricSinkValidationStep> validationLogger)
+            : base(validationLogger)
+        {
+            _runtimeConfiguration = runtimeConfiguration;
+            _metricsDeclarationProvider = metricsDeclarationProvider;
+        }
+
+        public string ComponentName { get; } = "Atlassian Statuspage Metric Sink";
+
+        public ValidationResult Run()
+        {
+            var currentRuntimeConfiguration = _runtimeConfiguration.Value;
+            var atlassianStatuspageConfiguration = currentRuntimeConfiguration?.MetricSinks?.AtlassianStatuspage;
+            if (atlassianStatuspageConfiguration == null)
+            {
+                return ValidationResult.Successful(ComponentName);
+            }
+
+            var errorMessages = new List<string>();
+            if (string.IsNullOrWhiteSpace(atlassianStatuspageConfiguration.PageId))
+            {
+                errorMessages.Add("No id of Atlassian Status page is configured");
```

Comment:
Just making this suggestion because it's a bit confusing since the product name is _Statuspage_, and you're validating the _pageId_ here, so I figure it's worth being specific.

```suggestion
                errorMessages.Add("No pageId for Atlassian Statuspage is configured");
```

Reply:
Good point, thanks!